.golangci.yml

```diff
@@ -63,6 +63,8 @@ linters-settings:
     exclude: ./.errcheck-exclude.txt
   errorlint:
     errorf: false # don't check whether fmt.Errorf uses %w to wrap errors
+  goconst:
```

Comment:
didn't know you could do this. i humorously included the same change in a different way here:

https://github.com/grafana/tempo/pull/3584/files#diff-6179837f7df53a6f05c522b6b7bb566d484d5465d9894fb04910dd08bb40dcc9R77-R81

i'm going to change mine to match yours. it does mean one of us may have to resolve the conflict.